boards/arm/stm32/nucleo-l152re/include/board.h

```diff
@@ -226,6 +226,4 @@
 #define GPIO_I2C1_SCL    GPIO_I2C1_SCL_2  /* PB8  CN5 pin 10, D15 */
 #define GPIO_I2C1_SDA    GPIO_I2C1_SDA_2  /* PB9  CN5 pin 9, D14 */
 
-/* I2C2 */
-
```

Comment:
let's merge the last two patch to the previous related patch instead.

Reply:
Do I need any other action to merge?

Reply:
git rebase --interactive HEAD~5
reorder the patch and change pick to squash in the editor
and git push -f again your branch.